protected/humhub/modules/admin/models/forms/DesignSettingsForm.php

```diff
@@ -291,12 +291,14 @@ public function load($data, $formName = null)
     /**
      * @return array a list of available themes
      */
-    public function getThemes()
+    public function getThemes(): array
     {
         $themes = [];
 
         foreach (ThemeHelper::getThemes() as $theme) {
```

Comment:
Maybe it's better when we implement this in the underlying "getThemes" method?

Reply:
@luke- Ok, then I think better is to do it in the method `ThemeHelper::getThemesByPath()` - commit https://github.com/humhub/humhub/pull/7683/commits/1d15b37c29cb83e6cf2e2f1d6f254614c9bd428b